client/src/components/utils/appHelper.js

```diff
@@ -146,10 +204,22 @@ const getDirections = (props, interactionTypes, place, dispatchFunction, errorHa
   props.changeInteractionType(interactionTypes.SELECTING_ROUTE);
 };
 
+/**
+ * Toggle Floating Action Button class.
```

Comment:
I like something a little more descriptive like "Toggle the proper class for the Floating Action Button"